lib/init/config-initializer.js

```diff
@@ -571,6 +572,15 @@ function promptUser() {
  *  @returns {void}
  */
 function initializeConfig() {
+    const argv = mri(process.argv.slice(2));
+
+    if (argv.config) {
+        const config = { extends: argv.config };
```

Comment:
It's a convention that the shared configs starting with `eslint-config`. can you just allow the user to omit it. e.g. `--config standard` to use `eslint-config-standard`?

Reply:
there is a function in `@eslint/eslintrc` to do so: https://github.com/eslint/eslintrc/blob/main/lib/shared/naming.js

Reply:
We won't need the `eslint-config` prefix now. I've used `getModulesList`, and `getModulesList` already handles this.

https://github.com/kecrily/create-config/blob/a20b5550194eb24208f0335796e177510ddb04a7/lib/init/config-initializer.js#L96

Reply:
well, I see. :)